app/controllers/admin/articles_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Admin
+  class ArticlesController < ApplicationController
+  	before_filter :admin_authorize!
+  	def index
+  		@article = Article.all
+    end		
```

Comment:
`end` at 6, 4 is not aligned with `def` at 4, 3<br>Trailing whitespace detected.